README.md

```diff
@@ -138,6 +139,56 @@ inspiration!
 
 If you want your template to appear in that list, just add the topic to it! 🏷
 
+## Show your support
+
+If you're using Copier, consider adding the Copier badge to your project's `README.md`:
+
+```md
+[![Copier](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-grayscale-inverted-border-orange.json)](https://github.com/copier-org/copier)
+```
+
+...or `README.rst`:
+
+```rst
+.. image:: https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-grayscale-inverted-border-orange.json
+    :target: https://github.com/copier-org/copier
+    :alt: Copier
+```
+
+...or, as HTML:
+
+```html
+<a href="https://github.com/copier-org/copier"
+    ><img
+        src="(https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-grayscale-inverted-border-orange.json"
+        alt="Copier"
+        style="max-width:100%;"
+/></a>
+```
```

Comment:
I think we should disable Prettier here and keep the HTML markup in one long line:


````suggestion
<!-- prettier-ignore-start -->
```html
<a href="https://github.com/copier-org/copier"><img src="https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-grayscale-inverted-border-orange.json" alt="Copier" style="max-width:100%;"/></a>
```
<!-- prettier-ignore-end -->
````

I also removed a `(` at the beginning of the `src` URL that didn't belong there.